cross/tcl/Makefile

```diff
@@ -2,33 +2,74 @@ PKG_NAME = tcl
 PKG_VERS = 8.6.6
 PKG_EXT = tar.gz
 PKG_DIST_NAME = $(PKG_NAME)$(PKG_VERS)-src.$(PKG_EXT)
-PKG_DIST_SITE = http://prdownloads.sourceforge.net/tcl/
+PKG_DIST_SITE = http://prdownloads.sourceforge.net/tcl
 PKG_DIR = $(PKG_NAME)$(PKG_VERS)/unix
 
 DEPENDS =
 BUILD_DEPENDS = cross/zlib
 
 HOMEPAGE = http://www.tcl.tk/
-COMMENT  = Tcl (Tool Command Language) is a very powerful but easy to learn dynamic programming language, suitable for a very wide range of uses, including web and desktop applications, networking, administration, testing and many more. Open source and business-friendly, Tcl is a mature yet evolving language that is truly cross platform, easily deployed and highly extensible.
-LICENSE  = http://tcl.tk/software/tcltk/license.html
+COMMENT = Tcl (Tool Command Language) is a very powerful but easy to learn dynamic programming language, suitable for a very wide range of uses, including web and desktop applications, networking, administration, testing and many more. Open source and business-friendly, Tcl is a mature yet evolving language that is truly cross platform, easily deployed and highly extensible.
+LICENSE = http://tcl.tk/software/tcltk/license.html
 
 GNU_CONFIGURE = 1
 
-# Prevent re-declaration of fixstrtod.o
-# export ac_cv_func_strtod=yes
-# export tcl_cv_strtod_buggy=1
+TCL_PACKAGE_PATH = /var/packages/$(PKG_NAME)/target
 
 # Enable 64-bit support in Tcl for 64-bit architectures
 ifneq ($(findstring $(ARCH), avoton braswell bromolow cedarview x64 x86),)
-	CONFIGURE_ARGS = tcl_cv_strtod_buggy=1 --prefix=$(STAGING_INSTALL_PREFIX) --enable-64bit
+	CONFIGURE_ARGS = tcl_cv_strtod_buggy=1 --prefix=$(TCL_PACKAGE_PATH) --enable-64bit
 else
-	CONFIGURE_ARGS = tcl_cv_strtod_buggy=1 --prefix=$(STAGING_INSTALL_PREFIX)
+	CONFIGURE_ARGS = tcl_cv_strtod_buggy=1 --prefix=$(TCL_PACKAGE_PATH)
 endif
 
+COMPILE_TARGET = myCompile
```

Comment:
Remove please, unnecessary with comment on L21 by referring to `STAGING_INSTALL_PREFIX` and setting INSTALL_PREFIX to the correct value (in SPK/Makefile, I've added a comment there as well)